### DISCUSSION TITLE
🥑 실뱌의 CSS 보물창고

### DISCUSSION THREAD

**Discussion by moolmin** [100-hours-a-week/discussions#69] [General]
(계속 추가예정)
## 애니메이션 관련
1. [Animista](https://animista.net/play/basic/scale-down): 종류 엄청 많고 코드도 제공
2. [react-spring](https://react-spring.dev/examples): 퀄리티 좋은 인터레티브 애니메이션
3. [framer motion](https://www.framer.com/motion/)
4. [css-tricks](https://css-tricks.com/tag/animation/): 신기하고 재밌는 애니메이션 있는곳
5. [animate](https://animate.style/): 애니메이션 깔끔

## 버튼 관련
1. [buttons cool](https://www.buttons.cool/buttons): 화려한거 많은데 사용이 간편하진 않음
2. [slider revolution](https://www.sliderrevolution.com/resources/css-button-hover-effects/): 버튼 호버 효과 모음
3. [uifresh](https://uifresh.net/css-button-style-animation-examples/): 버튼 호버 효과 많고 사용 간편

## 그림자/색상 관련
1. [CSS Scan](https://getcssscan.com/css-box-shadow-examples): 사용쉬움, 그림자 고급짐
2. [10015](https://10015.io/tools/css-gradient-generator): 그라데이션 css코드로 제공
3. [khroma](https://www.khroma.co/generator): 색상 조합 추천 사이트인데, 퀄리티 짱죠음
6. [Neumorphism.io](https://neumorphism.io/#e0e0e0): neumorphism 스타일 css 복붙만해서 쓰면됨

## Shape 관련
1. [말풍선 모양 css 제공](https://projects.verou.me/bubbly/)
2. [clip-path generator](https://bennettfeely.com/clippy/)

## 마우스 커서 관련
1. [react-cursorify](https://github.com/morethanmin/react-cursorify): 귀여운 손 이모지 커서도 있당
2. [react-animated-cursor](https://github.com/stephenscaff/react-animated-cursor): 심플한 원모양이 커서들. 클릭하면 동그라미 커져

## 스크롤 관련
1. [AOS](https://michalsnik.github.io/aos/)
3. [GSAP](https://gsap.com/scroll/) 

## 기타
1. [PX to REM converter](https://nekocalc.com/px-to-rem-converter)
4. [Pixel Me](https://pixel-me.tokyo/en/): 일반 이미지 픽셀화 해주는데 퀄리티 좋음
5. [Clipping Magic](https://ko.clippingmagic.com/): 누끼 세세하게 딸수있음
7. [shadcn](https://ui.shadcn.com/docs/components/carousel): 깔끔한 carousel
8. [text effect](https://wpdean.com/css-text-animation/)
9. [text effect2](https://www.sliderrevolution.com/resources/css-text-animation/)
10. [awwwards](https://www.awwwards.com/): 짱멋진 웹사이트 모음

**Comment by HidenLee** [comment]
꼭 필요했던 정보 감사합니다!!!

**Reply by collie-jun** [reply]
실뱌 채고

**Comment by ji006** [comment]
감사합니당~!!🙇🏻‍♀️

**Comment by yzooop** [comment]
ㅅㅂㅇㄴ ㅅㅇㅇㅇ

**Comment by mepwj** [comment]
실뱌 채고
